### PR TITLE
blockchain-carbon-accounting

### PR DESCRIPTION
**Pull request by opentaps**
CO2 emissions accounting calculations using Hyperledger Fabric

Thanks @C0rWin for looking through it.

### CODE REVIEW

**Review by C0rWin — APPROVED** [review]

File: labs/blockchain-carbon-accounting.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+For more details, see [Carbon Accounting and Certification Working Group](https://wiki.hyperledger.org/display/CASIG/Carbon+Accounting+and+Certification+Working+Group)
+
+# Initial Committers
+- https://github.com/opentaps
+- https://github.com/jwickers
+- https://github.com/rybalko
+- https://github.com/udosson
+
+# Sponsor
+_Provide the name of your sponsor. A sponsor is required and must be a maintainer of one of the Hyperledger projects, a TSC member, or a WG chair._
```
Comment (vipinsun) [inline labs/blockchain-carbon-accounting.md:28]: Remove this line which is left over from the template

**Review by vipinsun — CHANGES_REQUESTED** [review]
Please have BARTEM@il.ibm.com comment on this PR saying that he sponsored the PR-
Or is @C0rWin Artem Barger?
If you are bringing in an outside repository from opentaps, it needs DCO sign off as well. In case you do not have this DCO please look at documentation which discussed how to squash commit with a dco before moving code in.

**Comment by C0rWin** [comment]
> Or is @C0rWin Artem Barger?

Yes, I'm :)

File: labs/blockchain-carbon-accounting.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+---
+layout: default
+title: Lab Proposal Document
```
Comment (lehors) [inline labs/blockchain-carbon-accounting.md:3]: I believe this should be set to the lab's name instead.

**Review by lehors — APPROVED** [review]
I think it'd be good to explain the motivation for organizations to participate in such a system - is that to fulfill some legal obligations? because it will help improve their public image? something else?

This is however not a requirement for this proposal to get approved though. I'm fine with it.

**Review by vipinsun — APPROVED** [review]

**Comment by vipinsun** [comment]
Merging as requested changes have been done.

**Comment by opentaps** [comment]
Thanks @lehors!  We wrote about it on the wiki:
https://wiki.hyperledger.org/display/CASIG/Carbon+Accounting+and+Certification+Working+Group

If you'd like we can add it to the lab proposal as well.

-----
Si Chen
Open Source Strategies, Inc.

Video: Fighting Climate Change with Blockchain and Open Source
<https://youtu.be/NgxNWXa_IjE>

**Pull request merged**